crates/ruff_python_ast/src/source_code/generator.rs

```diff
@@ -1352,10 +1357,10 @@ impl<'a> Generator<'a> {
         self.p(brace);
         self.buffer += &generator.buffer;
 
-        if conversion.to_u32() != ConversionFlag::None as u32 {
+        if conversion != ConversionFlag::None {
```

Comment:
Nit: Could we add `is_none` helpers to `ConversionFlag`?

Reply:
Sure, actually it is already added in this version 😉

Reply:
I applied is_none here